<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowForm</class>
 <widget class="QDialog" name="MainWindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>323</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>458</width>
    <height>323</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>458</width>
    <height>323</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Marksheet Software</string>
  </property>
  <widget class="QListWidget" name="studentList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>321</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchField">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>19</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="searchLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>23</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Search:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>91</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="goButton">
      <property name="text">
       <string>Go!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="viewMarks">
      <property name="text">
       <string>View Marksheet</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Add">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Edit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Delete">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="statButton">
      <property name="text">
       <string>Stats.....</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>searchField</tabstop>
  <tabstop>goButton</tabstop>
  <tabstop>viewMarks</tabstop>
  <tabstop>Add</tabstop>
  <tabstop>Edit</tabstop>
  <tabstop>Delete</tabstop>
  <tabstop>statButton</tabstop>
  <tabstop>studentList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
